Dockerfile

```diff
@@ -233,7 +250,8 @@ COPY server/Makefile server/Makefile
 RUN cd server && \
     make gen-server && \
     pip install -r requirements_cuda.txt && \
-    pip install ".[attention, bnb, accelerate, compressed-tensors, marlin, moe, quantize, peft, outlines]" --no-cache-dir && \
+    pip install -r requirements_poetry.txt --no-cache-dir && \
```

Comment:
What's the issue again ? Why are we not using poetry directly ?

Also maybe we should take the opportunity to switch to `uv` ? https://docs.astral.sh/uv/pip/compile/#locking-requirements I have no idea, but it seems like the poetry lock isn't locking tightly enough, maybe ?